### ISSUE TITLE
[Feat]: Document the lack of IPv6 support in our install/update process, as well as workarounds.

### ISSUE THREAD

**Issue by Ferroin** [netdata/netdata#15877]
### Problem

Full explanation here: https://github.com/netdata/netdata/issues/15816#issuecomment-1683796566

TLDR: We can’t currently support IPv6 in the installer because GitHub are horrendously behind the times and don’t provide IPv6 service for more than 90% of their services. We have similar issues with the updater when not using native packages.

### Description

We should properly document this issue in our installer and updater documentation, as well as primary workarounds:

- For users who are able to use our native packages, manual setup of native package repos as outlined in our native package documentation is a viable workaround, since our repos do provide IPv6 access.
- For other users, NAT64, either provided by their ISP or through a public NAT64 gateway, is known to allow installation to work correctly.

### Importance

must have

### Value proposition

Less confusion for users on IPv6-only systems.

### Proposed implementation

Suggested location is in the primary kickstart install documentation, as that is the only method most users are likely to use that is affected by this issue (Docker is a non-issue, as both DockerHub and Quay provide IPv6 access, and our DEB/RPM repos do as well as mentioned above).

We should also mention this in the updater docs, as that requests GitHub endpoints that have this limitation as well.
